tests/test_authorization.py

```diff
@@ -14,7 +14,7 @@ def test_get_report_by_user(self):
         self.app.post('/api/v1/pmi', headers={"Content-Type":"application/json"}, data=json.dumps(downpayment_ten))
         self.app.post('/api/v1/pmi', headers={"Content-Type":"application/json"}, data=json.dumps(downpayment_fifteen))
     #Given
-        harry_uid = 'jhdfs783uy4ir98f7ygh2jeuwidsi8943yrih'
+        harry_uid = { "uid": 'jhdfs783uy4ir98f7ygh2jeuwidsi8943yrih' }
```

Comment:
Glad we caught this issue was in our test and not in the controller method itself